pkg/controller/storagecluster/cephcluster.go

```diff
@@ -51,10 +51,12 @@ func (r *ReconcileStorageCluster) ensureCephCluster(sc *ocsv1.StorageCluster, re
 		}
 	}
 
-	if sc.Spec.Network.IsMultus() {
-		err := validateMultusSelectors(sc.Spec.Network.Selectors)
-		if err != nil {
-			return err
+	if sc.Spec.Network != nil {
+		if sc.Spec.Network.IsMultus() {
+			err := validateMultusSelectors(sc.Spec.Network.Selectors)
+			if err != nil {
+				return err
+			}
```

Comment:
This is same as
https://github.com/openshift/ocs-operator/pull/746/files#diff-d59be10a9ed83fc2cbc7d93e0e430388R245-R248

Is it possible to add a common function? If later we forget that there are 2 places to make changes, it'll cause problems.

Reply:
done